python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion.py

```diff
@@ -101,17 +101,8 @@ async def complete_async(
             str -- The completed text.
         """
         prompt_to_message = [("user", prompt)]
-        chat_settings = ChatRequestSettings(
```

Comment:
What's the rationale for removing the ChatRequestSettings creation here? This method and `complete_stream_async` are here so that a chat model can be used as a few-shot text completion model like text-davinci-003. 

I agree that it's a bit wasteful to have the code here. It could be abstracted to `_send_chat_request` such that  `_send_chat_request` can accept either ChatRequestSettings or CompleteRequestSettings and does the conversion there. 

I could totally be missing something too.

Reply:
We realized that there is a difference between `complete..` and `complete_chat`, we will be adding it back soon.

Reply:
I have added back the original `ChatRequestSettings` creation, now with also addressing the `token_selection_biases` field. Thank you!